docs/docs/use-cases/day-2-operations.md

```diff
@@ -73,28 +74,14 @@ are executed before the updated pods are scheduled.
 In this case, the deployment should be changed as follows:
 
 ```yaml
-{% include "./assets/deployment-new-image-and-version.yaml" %}
+{% include "./assets/day-2-operations/deployment-new-image-and-version.yaml" %}
```

Comment:
in order to have the code block indented as well in the rendered page, you need to indent the whole code block here. please try out how many spaces are needed, it's not clear to me right now 😅

Reply:
4 spaces seemed to have done the trick :)